src/sagemaker_server.cc

```diff
@@ -628,6 +628,68 @@ SagemakerAPIServer::SageMakerMMEHandleInfer(
   }
 }
 
+bool
+SagemakerAPIServer::SageMakerMMEUnloadModelCheckStatus(const char* model_name)
```

Comment:
I don't love the name if it just returns a `bool`. Hard to tell without looking at the code what the bool represents. Does `true` mean it is unloaded? Does `true` mean the model is available, so not yet unloaded? I have no idea since I have not read the implementation code yet.

Reply:
Makes sense, I have changed the function name, and am no longer using a return type of bool